coral-component-taglist/src/scripts/TagList.js

```diff
@@ -293,6 +293,12 @@ const TagList = Decorator(class extends BaseFormField(BaseComponent(HTMLElement)
 
     // adds the role to support accessibility
     attachedItem.setAttribute('role', 'row');
+
+    // adds role to parent to support accessibility, if it doesn't already have it
+    if (attachedItem.parentElement.getAttribute('role') === null) {
```

Comment:
attachedItem.parentElement is coral-component-taglist right ?? You can use `this` in place